README.md

```diff
@@ -13,106 +13,50 @@ It builds with the latest stable Rust and supports `#[no_std]` targets.
 
 ```rust
 extern crate mos6502;
-
-use mos6502::cpu;
 use mos6502::address::Address;
+use mos6502::cpu;
 
 fn main() {
-    let mut cpu = cpu::CPU::new();
+    println!("Enter two numbers (< 128) to know their GCD:");
```

Comment:
I think we should just choose two numbers and remove this section here. `skeptic` seems to fail the unit tests because the input is empty. 😉

Reply:
I should remove it only from README?

Reply:
Yup. In case of the code in the `example` folder it makes a lot of sense to run the program multiple times with different inputs to see how it behaves. In case of the `README.md` however, we have to use something that doesn't require any user-interaction because [`rust-skeptic`](https://github.com/budziq/rust-skeptic) runs the code as an automated integration test (and we can't have any user input there).

Reply:
Yep